From 8653fe4af3f68acc6c11e8536238b2b061db8b81 Mon Sep 17 00:00:00 2001
From: "Radu Pirea (NXP OSS)" <radu-nicolae.pirea@oss.nxp.com>
Date: Tue, 5 Oct 2021 12:11:10 +0300
Subject: [PATCH 01/14] s32v234hpcsom: Disable pinctrl for UART1

The UART1 is used as a boot mode pin for other SoC.
So we disable it and use only UART0.

Issue: ALB-7928
Upstream-Status: Pending 

Signed-off-by: Radu Pirea (NXP OSS) <radu-nicolae.pirea@oss.nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 board/freescale/s32v234hpcsom/s32v234hpcsom.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/board/freescale/s32v234hpcsom/s32v234hpcsom.c b/board/freescale/s32v234hpcsom/s32v234hpcsom.c
index be434dd366..a6c8744d65 100644
--- a/board/freescale/s32v234hpcsom/s32v234hpcsom.c
+++ b/board/freescale/s32v234hpcsom/s32v234hpcsom.c
@@ -23,14 +23,6 @@ static void setup_iomux_uart(void)
 	writel(SIUL2_MSCR_PORT_CTRL_UART_RXD, SIUL2_MSCRn(SIUL2_MSCR_PA11));
 	/* set UART0 RXD - IMCR[200] - to link to PA11 */
 	writel(SIUL2_IMCR_UART_RXD_to_pad, SIUL2_IMCRn(SIUL2_IMCR_UART0_RXD));
-
-	/* set PA14 - MSCR[14] - for UART1 TXD*/
-	writel(SIUL2_MSCR_PORT_CTRL_UART_TXD, SIUL2_MSCRn(SIUL2_MSCR_PA14));
-
-	/* set PA13 - MSCR[13] - for UART1 RXD */
-	writel(SIUL2_MSCR_PORT_CTRL_UART_RXD, SIUL2_MSCRn(SIUL2_MSCR_PA13));
-	/* set UART1 RXD - IMCR[202] - to link to PA13 */
-	writel(SIUL2_IMCR_UART_RXD_to_pad, SIUL2_IMCRn(SIUL2_IMCR_UART1_RXD));
 }
 
 static void setup_iomux_i2c(void)
-- 
2.17.1

